.travis.yml

```diff
@@ -62,6 +63,7 @@ matrix:
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda.sh
+    - python -m pip install $ASTROPY_GIT
```

Comment:
What is the advantage of using `python -m` instead of `PIP_DEPENDENCIES`?

Reply:
I actually should have used `PIP_DEPENDENCIES` here. The reason for using it below is that `gwcs` needs to be installed with `--no-deps`, but `astropy-helpers` needs to install the dependencies. I don't think I can do this with `PIP_DEPENDENCIES` since it will use the same `pip` flags in both cases, which doesn't work. This is what was causing me so much trouble on the CI runs on a previous PR.

Reply:
Ah, good to know. Thanks!